vmdb/app/presenters/explorer_presenter.rb

```diff
@@ -105,8 +102,6 @@ def process
 
     @out << "dhxAccord.openItem('#{@options[:open_accord]}');" unless @options[:open_accord].to_s.empty?
 
-    @options[:trees_to_replace].each { |tree, opts| @out << replace_tree(tree,opts) }
```

Comment:
@chessbyte @mathewd 
- @options[:trees_to_replace].each { |tree, opts| @out << replace_tree(tree,opts) } this line of code was not in use, this code was specific to dhtmlxtrees, DhtmlxTrees were replaced by Dynatree.